_posts/2022-11-12-Aya-gsoc-final-report.md

```diff
@@ -0,0 +1,150 @@
+
+So, my [GSoC project] was about developing a new guessing API, that will make the guessing process more convenient and context-specific, plus developing the first two context-specific guessers, which are [PDB] Guesser and             [Martini forcefield] Guesser.
+
+
+### 1- Developing the new guesser API
+
+The first thing to start with was the development of the new guesser API and the building of the [DefaultGuesser] class (which will represent a general-purpose guesser that has the current general-purpose guesser methods). But before that, we must first remove all the usage of the old guessing methods inside the MDAnalysis library to prepare for the new stage of having context-aware guessers and the guessing API. But this can lead to unexpected wrong behavior if we did without being careful. Guessing mainly takes place inside 16 topology parsers (`types` and `mass` guessing specifically), and not all guessing inside those parsers happens the same way. So, to not break the current behavior, we must be careful with removing all the guessing from those parsers and yet keep the same default output while developing the new API.
+
+N.B.: it was important to have all the work on developing both the guesser API and the [DefaultGuesser] plus changing parsers behavior to be done inside one pull request here: [#3753]; because all thoses changes have a direct effect on each others, for example, the optmization of the new guesser API must be measured by how far the topology output changed after removing all the guessings processes from parsers. This was the most challenging thing about the project; as all these required changes are interconnected and must be done simultaneously, to see how it reflects and interacts with each other, and accordingly we can achieve optimization. So, working on this PR was not sequential as mentioned below, it was rather a continuous back-and-forth update on all touched parts of the library.
+
+#### - Removing types and mass guessing from Topology Parsers ([commit 0cbc497]) 
+First, I navigated through all parsers to see where and how types and mass guessing take place. The below table shows the behavior of types and masses guessing inside parsers.
+![Guessing inside Parsers](public/images/final_report_aya/parsers_guessing.png)
```

Comment:
change to
```suggestion
![Guessing inside Parsers]({{ site.baseurl }}{{ site.images }}/final_report_aya/parsers_guessing.png)
```
Otherwise the PNG will not be found.